packages/dashboard/src/components/iot-dashboard/iot-dashboard-internal.tsx

```diff
@@ -419,6 +456,17 @@ export class IotDashboardInternal {
             }}
           ></div>
         )}
+
+        <iot-dashboard-context-menu
+          messageOverrides={this.messageOverrides}
+          copyWidgets={() => this.onCopy()}
```

Comment:
is there a reason not to just pass in the funcdtions? i.e.

```copyWidgets={this.onCopy}```

rather than all these wrappers.

Also this made me think - we should probably just keep the naming consistent. i.e. `onDelete` rather than turning `onDelete => deleteWidgets`

Reply:
ya I suppose we can keep the naming the same. will update and just pass down the function